moldesign-examples/Untitled1.ipynb

```diff
@@ -0,0 +1,1913 @@
+{
```

Comment:
Please go ahead and move this notebook to `moldesign/_notebooks`, and then you can delete the rest of the `moldesign-examples` directory